tests/ci/install_nuocd.sh

```diff
@@ -5,13 +5,14 @@ export NUOCMD_API_SERVER=http://localhost:8888
 
 sudo cp -r nuocd /opt/.
 sudo chmod 777 /opt/nuocd
-pip install -r requirements.txt
+pip3 install -r requirements.txt
 
+sudo cp conf/telegraf.conf  /etc/telegraf/telegraf.conf
 sudo cp conf/nuodb.conf /etc/telegraf/telegraf.d
 sudo cp conf/outputs.conf /etc/telegraf/telegraf.d
 sudo cp bin/nuocd /usr/local/bin/nuocd
 
 sudo systemctl daemon-reload
 sudo systemctl stop telegraf
 
-/usr/bin/telegraf --config /etc/telegraf/telegraf.conf --config-directory /etc/telegraf/telegraf.d&
\ No newline at end of file
+/usr/bin/telegraf --config /etc/telegraf/telegraf.conf --config-directory /etc/telegraf/telegraf.d
```

Comment:
I think that this command blocks and that is why it was run in the background.
Have you tried to run the tests in bare-metal?

Reply:
The test in bare metal were failing and going in an infinite loop. For reference https://app.circleci.com/pipelines/github/nuodb/nuodb-collector/25/workflows/526a19c0-dd4a-4fe8-88f8-aab736aee108/jobs/33

Reply:
I'm confused about how this ever could have worked. The command puts the `telegraf` service in the background, but it does not invoke it with `nohup`, so I'm pretty sure it should will just be shutdown as soon as the script exits.